tests/test_zoom.py

```diff
@@ -14,19 +14,44 @@
 import unittest
 
 import numpy as np
+import torch
 from parameterized import parameterized
 from scipy.ndimage import zoom as zoom_scipy
 
 from monai.data import MetaTensor, set_track_meta
 from monai.transforms import Zoom
-from tests.utils import TEST_NDARRAYS_ALL, NumpyImageTestCase2D, assert_allclose, test_local_inversion
+from monai.transforms.lazy.functional import apply_transforms
+from tests.utils import (
+    DEFAULT_TEST_AFFINE,
+    TEST_NDARRAYS_ALL,
+    NumpyImageTestCase2D,
+    assert_allclose,
+    test_local_inversion,
+)
 
 VALID_CASES = [(1.5, "nearest"), (1.5, "nearest"), (0.8, "bilinear"), (0.8, "area")]
 
 INVALID_CASES = [((None, None), "bilinear", TypeError), ((0.9, 0.9), "s", ValueError)]
 
 
 class TestZoom(NumpyImageTestCase2D):
+    @parameterized.expand(VALID_CASES)
+    def test_pending_ops(self, zoom, mode):
+        im = MetaTensor(self.imt[0], meta={"a": "b", "affine": DEFAULT_TEST_AFFINE})
+        zoom_fn = Zoom(zoom=zoom, mode="bilinear", keep_size=False, dtype=torch.float64)
+        # non-lazy
+        expected = zoom_fn(im)
+        self.assertIsInstance(expected, MetaTensor)
+        # lazy
+        zoom_fn.lazy_evaluation = True
+        pending_result = zoom_fn(im)
+        self.assertIsInstance(pending_result, MetaTensor)
+        assert_allclose(pending_result.peek_pending_affine(), expected.affine)
+        assert_allclose(pending_result.peek_pending_shape(), expected.shape[1:])
+        result = apply_transforms(pending_result, mode="bilinear", dtype=np.float64, align_corners=True)[0]
```

Comment:
A little confused here, does it mean we only support {``"bilinear"``, ``"nearest"``} in mode and {``"zeros"``, ``"border"``, ``"reflection"``} in padding mode when we doing lazy resampling?
https://github.com/Project-MONAI/MONAI/blob/bf55f22c461bac286e3078a242d9d0b90a43ba00/monai/transforms/lazy/functional.py#L46-L52

Reply:
We support all the modes that pytorch grid_sample and scipy map_coordinates support. But the underlying implementations are different, we can't guarantee exactly the same results using these different backends/precisions/devices. This test is only to check that we are using the resampling APIs correctly.